src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.db;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    public static final Map<String, Integer> report = new HashMap<>();
+}
```

Comment:
```suggestion
    public void add(String fruit, int quantity) {
    }

    public int getQuantity(String fruit) {
    }

    public void remove(String fruit, int quantity) {
    }
}
```
you can do smth like this